### ISSUE TITLE
docs: provide pointers to enable CORS

### ISSUE THREAD

**Issue by paoloantinori** [Apicurio/apicurio-registry#707]
https://quarkus.io/guides/http-reference#cors-filter

**Comment by EricWittmann** [comment]
Naive CORS is enabled by default in Registry, but we haven't done anything to document these extra features.  I'm assuming you are suggesting we link to the additional CORS configuration from the Registry docs, yes?  Thoughts @smccarthy-ie ?

**Comment by paoloantinori** [comment]
yes. As much as this is implemented by a third party component we use, you could say that it's also part of our own "API" so we should at minumum mention where to find the corresponding documentation or, even better, provide some guidance directly in our own docs.

**Comment by smccarthy-ie** [comment]
@EricWittmann @paoloantinori Sorry folks missed this one in my mail filters! :)  Sure, this sounds like a candidate for the user docs. 

@EricWittmann Can you give an example of how you can enable CORS config in Apicurio Registry? For example, are there corresponding registry env vars for each CORS setting? Thanks.

**Comment by carlesarnal** [comment]
Hi @smccarthy-ie, the way to enable CORS in Apicurio Registry is the quarkus default one, you have all the info [here](https://quarkus.io/guides/http-reference#cors-filter).

**Comment by smccarthy-ie** [comment]
OK cool, thanks @carlesarnal :)
